r_scripts/data_wrangle/asthma_cohort.R

```diff
@@ -0,0 +1,122 @@
+# ------------------------------------------------------------------------------
```

Comment:
give this a new title and update the information inside the header. Remove any comments that aren't relevant
anymore.